tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -258,6 +294,13 @@ export class GraphExecutor implements FunctionExecutor {
       }
       return outputs.map(name => getTensor(name, tensorsMap, context));
     });
+
+    if (this.dumpMode === DumpMode.Sync) {
+      this.tensorsMap = tensorsMap;
+    } else {
+      this.tensorsMap = null;
```

Comment:
It's safe to say `this.tensorsMap = tensorsMap` since L280 make sure that async won't go to this path?

Reply:
Yes. This 'if' help to remind this is dump specific. Under non-dump mode, this.tensorsMap will never be used.